pyatrea/__init__.py

```diff
@@ -146,16 +328,11 @@ def setPower(self, power):
             return False
         power -= 1
 
-        if(power < 99):
-            power = "0"+str(power)
-        elif(power < 10) and (power >= 0):
-            power = "00"+str(power)
-        elif(power == 100):
-            power = str(power)
-        else:
+        if(power < 0 or power > 100):
             return False
 
-        self.commands['H10708'] = "00"+power
+        self.setCommand('H10708', power)
+        self.setCommand('H01020', power)
```

Comment:
These are the indexes that get changed on setPower when I change from 50 to 67 via UI:

```
{('H10714', '50'), ('I00009', '40'), ('I12608', '5000'), ('I12609', '5000'), ('I00009', '41'), ('H10200', '6700'), ('H10714', '67'), ('H10201', '5000'), ('H10200', '5000'), ('H10201', '6700'), ('H10704', '50'), ('H10704', '67'), ('H10708', '50'), ('H10708', '67'), ('I12608', '6700'), ('I12609', '6700')}
```

*Ignore I00009, seems to be a counter of requests made.

None of these is in params[ids] rendering setCommand not working.

I suggest adding check if id is in status, and if yes, execute it too.